media-fonts/ibm_plex/ibm_plex-0.5.3.recipe

```diff
@@ -0,0 +1,34 @@
+SUMMARY="The package of IBM’s new typeface, IBM Plex"
+DESCRIPTION="Thanks for your interest in IBM Plex, IBM’s new corporate type family. \
+We welcome your feedback and contributions to this project. \
+IBM Plex is open source, and should be used by IBMers for all typographical situations, \
+replacing Helvetica Neue, whenever possible."
+HOMEPAGE="https://ibm.github.io/type"
+COPYRIGHT="2017 IBM Corp. with Reserved Font Name 'Plex'"
+LICENSE="SIL Open Font License v1.1"
+REVISION="1"
+SOURCE_URI="https://github.com/IBM/type/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="0e212de6d0f33604b6c22e9743aab05bc12d482b1bdc7402972f3f6a9aa92a77"
+SOURCE_DIR="type-$portVersion"
+
+ARCHITECTURES="any"
+DISABLE_SOURCE_PACKAGE=yes
+
+PROVIDES="ibm_plex = $portVersion"
+REQUIRES=""
+BUILD_REQUIRES=""
```

Comment:
Please add a newline here as required by our [Guidelines][0]

```diff
 REQUIRES=""
+
 BUILD_REQUIRES=""
```

[0]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering

Reply:
Sorry, I'm making a lot of exceptions :)

Hmm, on at least one other package, noboto, to be precise, we don't even define `REQUIRES` and `BUILD_REQUIRES`, so I'd say that these variables are fine even if they touch themselves.

Reply:
So maybe we should remove all empty variables